docs/making_transfers.md

```diff
@@ -50,6 +60,40 @@ fetch('https://api.tez.ie/keys/hangzhounet/', {
   .catch((error) => render(`Error: ${JSON.stringify(error, null, 2)}`));
 ```
 
+</TabItem>
+  <TabItem value="walletAPI">
+
+```js live noInline wallet
+// import { TezosToolkit } from '@taquito/taquito';
+// const Tezos = new TezosToolkit('https://hangzhounet.api.tez.ie');
+
+render(`Fetching a private key...`);
+fetch('https://api.tez.ie/keys/hangzhounet/', {

```

Comment:
No need to fetch a private key when using the wallet

```
// import { TezosToolkit } from '@taquito/taquito';
// const Tezos = new TezosToolkit('https://hangzhounet.api.tez.ie');

    const amount = 0.5;
    const address = 'tz1h3rQ8wBxFd8L9B3d7Jhaawu6Z568XU3xY';

    render(`Transfering ${amount} ꜩ to ${address}...`);
    return Tezos.wallet.transfer({ to: address, amount: amount }).send()
.then((op) => {
    render(`Waiting for ${op.opHash} to be confirmed...`);
    return op.confirmation(1).then(() => op.opHash);
  })
  .then((hash) => render(`Operation injected: https://hangzhou.tzstats.com/${hash}`))
  .catch((error) => render(`Error: ${JSON.stringify(error, null, 2)}`));
```

Reply:
changed